assets/js/persistence.js

```diff
@@ -3,14 +3,12 @@
     populateSecondaryGeneratorList, isSubset, handleNewCurrentLang */
 /* global srcLangs:true, dstLangs:true, recentSrcLangs: true, recentDstLangs:true, curSrcLang:true, curDstLang:true, locale:true */
 
-var URL_PARAM_Q_LIMIT = 1300,
```

Comment:
This shouldn't be configurable either. The things that are technical limitations and don't make sense to configure, shouldn't be configurable.

Reply:
But this limitation is not technical, it's practical.

Reply:
No, it's technical. Some browsers have limitations on maximum URL length.

Reply:
Yeah, but there is no definite maximum. I'll revert this anyway.